src/palette/internal/palettecreator.cpp

```diff
@@ -599,11 +599,17 @@ PalettePtr PaletteCreator::newFingeringPalette(bool defaultPalette)
         f->setXmlText(QString(finger[i]));
         sp->appendElement(f, QT_TRANSLATE_NOOP("palette", "LH guitar fingering %1"));
     }
-    finger = "0123456";
-    for (unsigned i = 0; i < strlen(finger); ++i) {
+
+    String stringNumber;
+    if (defaultPalette) {
+        stringNumber = "123456";
+    } else {
+        stringNumber = "0123456789";
+    }
+    for (unsigned i = 0; i < stringNumber.size(); ++i) {
```

Comment:
Theoretically you could still just do `finger = defaultPalette ? "123456" : "0123456789";`…

Reply:
Oh right, that went over my head 😁